build_projects/shared-build-targets-utils/Utils/BuildVersion.cs

```diff
@@ -7,7 +7,7 @@ public class BuildVersion : Version
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
-        public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-rc3-{CommitCountString}";
+        public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-{ReleaseSuffix}-{CommitCountString}";
         public string ProductionVersion => $"{Major}.{Minor}.{Patch}";
```

Comment:
@weshaggard - can we get this fixed??  We've been talking about "making this better" ever since it was first introduced, and we still have these problems with the way the versions are passed into the `Microsoft.NETCore.App` package build.